libraries/engage/components/SheetList/index.jsx

```diff
@@ -54,7 +54,9 @@ class SheetList extends Component {
 
           const classes = [styles.item];
 
-          if (!isLast) {
+          if (isSelected) {
```

Comment:
The `itemNotLast` takes care, that there is no separator at the bottom of the last list entry. When the last item is selected, this separator now re-appears.

![Bildschirmfoto 2019-06-24 um 09 24 24](https://user-images.githubusercontent.com/13087792/59999352-0d8fcc80-9662-11e9-8320-e4115e414dbe.png)